packages/remix-dev/cli/run.ts

```diff
@@ -1,11 +1,12 @@
 import * as path from "path";
-import meow from "meow";
 import inspector from "inspector";
+import meow from "meow";
 import inquirer from "inquirer";
-// import chalkAnimation from "chalk-animation";
```

Comment:
Could you remove this dep from the lockfile and package.json so we can close #2547?